client/gutenberg/extensions/mailchimp/edit.jsx

```diff
@@ -144,12 +144,13 @@ class MailchimpSubscribeEdit extends Component {
 						'You need to connect your Mailchimp account and choose a list in order to start collecting Email subscribers.'
 					) }
 					<br />
-					<ExternalLink href={ connectURL }>{ __( 'Set up Mailchimp form' ) }</ExternalLink>
 					<br />
-					<br />
-					<Button isDefault onClick={ this.apiCall }>
-						{ __( 'Re-check Connection' ) }
+					<Button isPrimary href={ connectURL } target="_blank">
+						{ __( 'Set up Mailchimp form' ) }
 					</Button>
+					<br />
+					<br />
+					<Button onClick={ this.apiCall }>{ __( 'Re-check Connection' ) }</Button>
```

Comment:
Could we switch to `ExternalLink` so we have a clear difference between the main action and the 2nd one.

Reply:
I opted not to use `ExternalLink` because the diagonal arrow seemed misleading, since this is no longer a link to an external site. To me, the arrow clearly indicates you will be taken to a new page. Disagree?

Reply:
Oh ok I see. In that case, can we just use a link?

Reply:
@thomasguillot did you mean `<Button is-borderless>` because this still doesn't have `href`? :-)

Reply:
@simison yep 😊

Reply:
Nice, I didn't know about `isBorderless` - added in https://github.com/Automattic/wp-calypso/pull/31089/commits/e2f0c0da4b808f7d98773be6636450c742adeaca. The focus state still looks button-like, though:

<img width="543" alt="screen shot 2019-03-01 at 7 57 25 am" src="https://user-images.githubusercontent.com/1477002/53639620-b750d680-3bf7-11e9-8583-a6fa1253440d.png">

Reply:
Button has a few tricks it can do: https://wpcalypso.wordpress.com/devdocs/client/components/button/README.md

Reply:
....aaand sorry, I'm referring Calypso button all the time here. :-) We're of course talking about core-button.

Reply:
@jeffersonrabb Humm ok that's an issue then.

Let's have "Set up Mailchimp form" as a `isPrimary isLarge` (sorry about the waste of time)
and "Re-check Connection" as a `isDefault isLarge`

Reply:
@simison looks like Gutenberg button doesn't support `isBorderless`: https://github.com/WordPress/gutenberg/tree/master/packages/components/src/button

and @thomasguillot no waste of time at all! Changes made in https://github.com/Automattic/wp-calypso/pull/31089/commits/9283023955d83cd602c21d21981a4ea0d51462e5

Reply:
Looks like its `isLink` in core:

https://github.com/WordPress/gutenberg/tree/535e0752051957f3e6ab7aca2e71e99ceffc9a6f/packages/components/src/button#readme

Reply:
Oh I like a `isLink` for "Re-check Connection". That way we can keep `isDefault isLarge` for "Set up Mailchimp form" and keep the same pattern as Core for placeholders.

Reply:
So for re-check connection, `isLink` and what else? `isLink` alone still has a button-esque focus state:

<img width="570" alt="screen shot 2019-03-01 at 8 28 35 am" src="https://user-images.githubusercontent.com/1477002/53641144-0f89d780-3bfc-11e9-8d72-65b062345076.png">

Also, while we're at it, should I make any change to the `Manage Connection` button in the sidebar? That's an `<ExternalLink/>` too, although it's identical in function to the primary button in the placeholder.

Reply:
"Re-check Connection" is fine like this. I like the fact that it's a link and now we can easily see the main action.

As for "Set up Mailchimp form", I'd switch back to `isDefault` rather than `isPrimary`.

I think the external link in the sidebar is fine, no need to worry about this 👍

Reply:
In https://github.com/Automattic/wp-calypso/pull/31089/commits/13f7ed56b1e69149a8389d35626fa96051a0a5e1 we now have `isDefault isLarge` for the Set up button, and `isLink` for Re-check. Sidebar unchanged.

Reply:
Amazing, thank you @jeffersonrabb